podpac/core/interpolation/interpolation_manager.py

```diff
@@ -594,13 +594,18 @@ def interpolate(self, source_coordinates, source_data, eval_coordinates, output_
         # TODO handle stacked issubset of unstacked case
         #      this case is currently skipped because of the set(eval_coordinates) == set(source_coordinates)))
         if eval_coordinates.issubset(source_coordinates) and set(eval_coordinates) == set(source_coordinates):
-            try:
-                output_data.data[:] = source_data.interp(output_data.coords, method="nearest").transpose(
-                    *output_data.dims
-                )
-            except (NotImplementedError, ValueError):
-                output_data.data[:] = source_data.sel(output_data.coords).transpose(*output_data.dims)
-            return output_data
+            if any(isinstance(c, StackedCoordinates) and c.ndim > 1 for c in eval_coordinates.values()):
+                # TODO AFFINE
```

Comment:
Not exactly. If you remove this case, the tests will fail because neither the `interp` nor the `sel` work with shaped `StackedCoordinates` (or `AffineCoordinates`, which are a special case of shaped StackedCoordinates). So we should implement this case.

But, it "works" currently because this acts to skip the short-circuit/optimization in this case and it falls through to the interpolation. So we should implement this optimization case, but it's less important than the real interpolation case.

We do need to make sure that we are testing both the short-circuit and real interpolation.

Does that make sense?

Reply:
I've now thought about this case for a while... I think it's actually pretty tricky to cover all the cases of subsets (non-strided subsets or random access) and it's best to leave it to the interpolation machinery. We could add a nearest neighbor interpolator here so that we avoid the trouble of going to a (for example) bilinear interpolation, but I don't think it's worth the effort. I'm happy to use xarray machinery, but that currently can't handle multi-indexes. So I think it's fine just to pass here and let the interpolation machinery take over. 

What do you think @jmilloy